plugins/woocommerce-admin/client/products/product-form-actions.tsx

```diff
@@ -14,6 +14,11 @@ import { registerPlugin } from '@wordpress/plugins';
 import { useFormContext } from '@woocommerce/components';
 import { Product } from '@woocommerce/data';
 import { recordEvent } from '@woocommerce/tracks';
+import { useSelect } from '@wordpress/data';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore No types for this exist yet.
+// eslint-disable-next-line @woocommerce/dependency-group
+import { store } from '@wordpress/viewport';
```

Comment:
I tried to add types to this, but the dependency did not work as expected.  Worth sanity checking to make sure it's not just my environment.